src/PostgREST/Types.hs

```diff
@@ -163,7 +163,13 @@ operators = M.fromList [
   ("is", "IS"),
   ("@@", "@@"),
   ("@>", "@>"),
-  ("<@", "<@")]
+  ("<@", "<@"),
+  ("&&", "&&"),
```

Comment:
Yeah, as I mentioned in the gitter chat, it requires the client to encode it as `%26`. I've found that most libraries do this already so it's not that inconvenient but it will need to be noted in the documentation.

In terms of replacing the ampersand with another character, the best ASCII equivalent I can think of is `e`since an `&` is written as that letter in handwriting. It looks kind of confusing though. It's also legal to use unicode characters in urls, so I suppose mathematical symbols like the intersection symbol could be used, but this is possibly very unfriendly.